metasploit-framework.gemspec

```diff
@@ -142,6 +142,7 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency 'bcrypt_pbkdf'
   spec.add_runtime_dependency 'ruby_smb', '~> 3.0'
   spec.add_runtime_dependency 'net-ldap'
+  spec.add_runtime_dependency 'net-smtp'
```

Comment:
Fixing the startup crash caused by net-smtp no longer being part of Ruby by default

> > The following default gems are now bundled gems. You need to add the following libraries to Gemfile under the > bundler environment.
> >   - net-ftp 0.1.3
> >  - net-imap 0.2.2
> >  - net-pop 0.1.1
> >  - net-smtp 0.3.1
> 
> https://www.ruby-lang.org/en/news/2021/12/25/ruby-3-1-0-released/

```
/home/user/.rvm/gems/ruby-3.1.1@metasploit-framework/gems/zeitwerk-2.5.4/lib/zeitwerk/kernel.rb:35:in `require': cannot load such file -- net/smtp (LoadError)
	from /home/user/.rvm/gems/ruby-3.1.1@metasploit-framework/gems/zeitwerk-2.5.4/lib/zeitwerk/kernel.rb:35:in `require'
	from /mnt/hgfs/metasploit-framework/lib/rex/proto/sms/model.rb:29:in `<top (required)>'
	from /home/user/.rvm/gems/ruby-3.1.1@metasploit-framework/gems/zeitwerk-2.5.4/lib/zeitwerk/kernel.rb:27:in `require'
	from /home/user/.rvm/gems/ruby-3.1.1@metasploit-framework/gems/zeitwerk-2.5.4/lib/zeitwerk/kernel.rb:27:in `require'
	from /home/user/.rvm/gems/ruby-3.1.1@metasploit-framework/gems/zeitwerk-2.5.4/lib/zeitwerk/loader/helpers.rb:95:in `const_get'
	from /home/user/.rvm/gems/ruby-3.1.1@metasploit-framework/gems/zeitwerk-2.5.4/lib/zeitwerk/loader/helpers.rb:95:in `cget'
	from /home/user/.rvm/gems/ruby-3.1.1@metasploit-framework/gems/zeitwerk-2.5.4/lib/zeitwerk/loader.rb:237:in `block (2 levels) in eager_load'
	from /home/user/.rvm/gems/ruby-3.1.1@metasploit-framework/gems/zeitwerk-2.5.4/lib/zeitwerk/loader/helpers.rb:26:in `block in ls'
	from /home/user/.rvm/gems/ruby-3.1.1@metasploit-framework/gems/zeitwerk-2.5.4/lib/zeitwerk/loader/helpers.rb:18:in `each_child'
	from /home/user/.rvm/gems/ruby-3.1.1@metasploit-framework/gems/zeitwerk-2.5.4/lib/zeitwerk/loader/helpers.rb:18:in `ls'
	from /home/user/.rvm/gems/ruby-3.1.1@metasploit-framework/gems/zeitwerk-2.5.4/lib/zeitwerk/loader.rb:232:in `block in eager_load'
	from /home/user/.rvm/gems/ruby-3.1.1@metasploit-framework/gems/zeitwerk-2.5.4/lib/zeitwerk/loader.rb:217:in `synchronize'
	from /home/user/.rvm/gems/ruby-3.1.1@metasploit-framework/gems/zeitwerk-2.5.4/lib/zeitwerk/loader.rb:217:in `eager_load'
	from /home/user/.rvm/gems/ruby-3.1.1@metasploit-framework/gems/zeitwerk-2.5.4/lib/zeitwerk/loader.rb:317:in `each'
	from /home/user/.rvm/gems/ruby-3.1.1@metasploit-framework/gems/zeitwerk-2.5.4/lib/zeitwerk/loader.rb:317:in `eager_load_all'
	from /home/user/.rvm/gems/ruby-3.1.1@metasploit-framework/gems/railties-6.1.5/lib/rails/application/finisher.rb:133:in `block in <module:Finisher>'
	from /home/user/.rvm/gems/ruby-3.1.1@metasploit-framework/gems/railties-6.1.5/lib/rails/initializable.rb:32:in `instance_exec'
	from /home/user/.rvm/gems/ruby-3.1.1@metasploit-framework/gems/railties-6.1.5/lib/rails/initializable.rb:32:in `run'
	from /home/user/.rvm/gems/ruby-3.1.1@metasploit-framework/gems/railties-6.1.5/lib/rails/initializable.rb:61:in `block in run_initializers'
	from /home/user/.rvm/rubies/ruby-3.1.1/lib/ruby/3.1.0/tsort.rb:228:in `block in tsort_each'
	from /home/user/.rvm/rubies/ruby-3.1.1/lib/ruby/3.1.0/tsort.rb:350:in `block (2 levels) in each_strongly_connected_component'
	from /home/user/.rvm/rubies/ruby-3.1.1/lib/ruby/3.1.0/tsort.rb:431:in `each_strongly_connected_component_from'
	from /home/user/.rvm/rubies/ruby-3.1.1/lib/ruby/3.1.0/tsort.rb:349:in `block in each_strongly_connected_component'
	from /home/user/.rvm/rubies/ruby-3.1.1/lib/ruby/3.1.0/tsort.rb:347:in `each'
	from /home/user/.rvm/rubies/ruby-3.1.1/lib/ruby/3.1.0/tsort.rb:347:in `call'
	from /home/user/.rvm/rubies/ruby-3.1.1/lib/ruby/3.1.0/tsort.rb:347:in `each_strongly_connected_component'
	from /home/user/.rvm/rubies/ruby-3.1.1/lib/ruby/3.1.0/tsort.rb:226:in `tsort_each'
	from /home/user/.rvm/rubies/ruby-3.1.1/lib/ruby/3.1.0/tsort.rb:205:in `tsort_each'
	from /home/user/.rvm/gems/ruby-3.1.1@metasploit-framework/gems/railties-6.1.5/lib/rails/initializable.rb:60:in `run_initializers'
	from /home/user/.rvm/gems/ruby-3.1.1@metasploit-framework/gems/railties-6.1.5/lib/rails/application.rb:391:in `initialize!'
	from /home/user/.rvm/gems/ruby-3.1.1@metasploit-framework/gems/railties-6.1.5/lib/rails/railtie.rb:207:in `public_send'
	from /home/user/.rvm/gems/ruby-3.1.1@metasploit-framework/gems/railties-6.1.5/lib/rails/railtie.rb:207:in `method_missing'
	from /mnt/hgfs/metasploit-framework/config/environment.rb:4:in `<top (required)>'
	from /mnt/hgfs/metasploit-framework/lib/msfenv.rb:17:in `require'
	from /mnt/hgfs/metasploit-framework/lib/msfenv.rb:17:in `<top (required)>'
	from ./msfconsole:18:in `require'
	from ./msfconsole:18:in `<main>'
```